PowerEditor/src/Parameters.cpp

```diff
@@ -3680,7 +3679,9 @@ void NppParameters::writeSession(const Session & session, const wchar_t *fileNam
 				(fileNameNode->ToElement())->SetAttribute(L"tabColourId", static_cast<int32_t>(viewSessionFiles[i]._individualTabColour));
 				(fileNameNode->ToElement())->SetAttribute(L"RTL", viewSessionFiles[i]._isRTL ? L"yes" : L"no");
 				(fileNameNode->ToElement())->SetAttribute(L"tabPinned", viewSessionFiles[i]._isPinned ? L"yes" : L"no");
-				(fileNameNode->ToElement())->SetAttribute(L"untitleTabRenamed", viewSessionFiles[i]._isUntitledTabRenamed ? L"yes" : L"no");
+				// Save this info only when it's an untitled entry
+				if (viewSessionFiles[i]._isUntitledTabRenamed)
+					(fileNameNode->ToElement())->SetAttribute(L"untitleTabRenamed", viewSessionFiles[i]._isUntitledTabRenamed ? L"yes" : L"no");

```

Comment:
Should be:
(fileNameNode->ToElement())->SetAttribute(L"untitleTabRenamed", L"yes");